.pathplanner/settings.json

```diff
@@ -9,9 +9,9 @@
     "to shooting pos"
   ],
   "autoFolders": [],
-  "defaultMaxVel": 4.5,
-  "defaultMaxAccel": 8.0,
-  "defaultMaxAngVel": 740.0,
-  "defaultMaxAngAccel": 1000.0,
```

Comment:
Where these lowered empirically?

Reply:
Note sure what that means, but we did some trials to measure our max acceleration and speed for both rotation and movement